src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -569,26 +577,26 @@ public ClusterDaemon(ClusterSettings settings)
 
             Context.ActorOf(Props.Create<ClusterHeartbeatReceiver>().WithDispatcher(settings.UseDispatcher), "heartbeatReceiver");
 
+            Receive<InternalClusterAction.GetClusterCoreRef>(msg => _coreSupervisor.Forward(msg));
```

Comment:
Removed `PatternMatch` and changed this to a `ReceiveActor`. Going to be doing that periodically as I touch the built-in cluster actors.